cmd/containerboot/main.go

```diff
@@ -565,6 +576,13 @@ runLoop:
 						log.Fatalf("storing device IPs and FQDN in Kubernetes Secret: %v", err)
 					}
 				}
+
+				if cfg.HealthCheckAddrPort != "" {
+					h.Lock()
+					h.hasAddrs = len(addrs) != 0
```

Comment:
Alternatively, we could consider the node ready when it has endpoints. I am not sure at the moment what would be the best, so used tailnet addresses- we could change it in the future when we get some feedback